src/cascadia/TerminalSettingsModel/CascadiaSettings.h

```diff
@@ -237,14 +238,13 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
     public:
         FragmentProfileEntry(winrt::guid profileGuid, hstring json) :
             _profileGuid{ profileGuid },
-            _json{ json } {}
+            _Json{ json } {}
 
         winrt::guid ProfileGuid() const noexcept { return _profileGuid; }
-        hstring Json() const noexcept { return _json; }
+        WINRT_PROPERTY(hstring, Json);

```

Comment:
Aren't we using `til::property` now or was that for something different? I forgot. 😅